test/e2e/apimachinery/garbage_collector.go

```diff
@@ -843,9 +839,9 @@ var _ = SIGDescribe("Garbage collector", func() {
 
 	// TODO: should be an integration test
 	/*
-		    Testname: garbage-collector-dependency-cycle
-		    Description: Ensure that a dependency cycle will
-			not block the garbage collector.
+		Release : v1.9
+		Testname: Garbage Collector, dependency cycle
+		Description: Create three pods, patch them with Owner references such that pod1 has pod3, pod2 has pod1 and pod3 has pod2 as owner references respectively. Delete pod1 SHOULD delete all pods. The dependency cycle SHOULD not block the garbage collection.
```

Comment:
s/SHOULD/MUST/ in both spots, I think

Reply:
@duglin Thanks for the review, I fixed as per your comments.